src/main/java/uk/gov/hmcts/darts/cache/token/service/Token.java

```diff
@@ -50,18 +59,29 @@ public String getId() {
         return TokenRegisterable.CACHE_PREFIX + ":" + tokenString + ":" + sessionId;
     }
 
-    public boolean valid() {
+
+    /**Ω
```

Comment:
Not sure what's happened with the java doc here, but there is some funny character after the asterisks